doc/getting-started/apl-nuc.rst

```diff
@@ -364,6 +364,74 @@ For hugeTLB to work, you'll need to reserve huge pages:
      by using a 2M page, but make sure your huge page reservation size is
      large enough for your usage.
 
+USB Device Sharing
+==========================================
+
+The ACRN hypervisor supports USB device sharing.  Suppose you have
+two keyboards and mice connected to your device, one keyboard and 
+mouse set for the SOS, and the other set for the UOS.  
+
+1. Boot the SOS and plug in the two keyboards and two mice
```

Comment:
It still bothers me that you say to plug in all four devices here, and then two steps later you say to plug them in one by one.  How about adding a note as part of this first step:

    .. note::  You may need to plug one device in and run ``dmesg`` to display
       the device info, then plug in the second device and run ``dmesg`` again, 
       until you have all devices plugged in and have noted the device info
       specific for each device.